src/main/kotlin/com/kruthers/gamemode4core/modes/ModMode.kt

```diff
@@ -52,6 +54,18 @@ class ModMode {
                 }
             }
 
+            // Clear potion effect
+            for (effect in player.activePotionEffects) {
+                player.removePotionEffect(effect.type)
+            }
+
+            // Load their build mode effects
+            playerData.getList("storage.mod_mode.build_data.effects")?.forEachIndexed { _, it ->
```

Comment:
this can just be `?.forEach { }` you wont need to define `it` either

Reply:
I'm regretting making 2 PRs at the same time haha.  Hopefully this merges properly with #3 being accepted first?  Otherwise if it's easier, I can close #3 and just add the change here